src/shared/ui/survey/Slider/SurveySlider.tsx

```diff
@@ -10,45 +10,44 @@ const SurveySlider: FC<SliderProps> = ({ config, ...props }) => {
   const {
     leftTitle,
     rightTitle,
-    items,
-    showTitles = true,
+    minValue,
+    maxValue,
     showTickMarks = true,
     showTickLabels = true,
     isContinuousSlider = false,
     leftImageUrl,
     rightImageUrl,
   } = config;
-
   const { onChange, onRelease, onPress, initialValue = 0 } = props;
 
   const onValueChange = (arrayOfValues: number[]) => {
-    const [firstElement] = items;
     const [value] = arrayOfValues;
-    const numericValue = value + Number(firstElement.value);
-    const roundedValue = Math.round(numericValue * 100) / 100;
+    const roundedValue = Math.round(value * 100) / 100;
     onChange(roundedValue);
   };
 
-  const initialIndex = items.findIndex(({ value }) => value === initialValue);
+  const length = maxValue - minValue;
+  const items = new Array(length).fill(1);
 
   return (
     <YStack>
       <Slider
-        defaultValue={[initialIndex !== -1 ? initialIndex : -100]}
+        defaultValue={[initialValue]}
         onResponderRelease={onRelease}
         onResponderStart={onPress}
         onValueChange={onValueChange}
         size={THUMB_SIZE}
-        max={items.length - 1}
+        max={maxValue - 1}
+        min={minValue}
         step={isContinuousSlider ? 0.01 : 1}
       />
 
       <XStack jc="space-between" mt={9}>
-        {items.map(({ value }) => {
+        {items.map((value, index) => {
```

Comment:
Please fill necessary values into items array (i.o 1) and use value, remove index